hack/bump-prow-images.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+latest_tag="$( gcloud container images list-tags gcr.io/k8s-prow/plank --format='value(tags)' --limit 1 | grep -Po "v[0-9]+\-[a-z0-9]+" )"
```

Comment:
tiny doc in here explaining what this is doing

Reply:
naaaah

Reply:
what's a doc?